src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/EnergyDistribution/src/law2Int.hpp

```diff
@@ -0,0 +1,5 @@
+static int law2Int( LAWS& law ){
+  constexpr static std::array< int, 14 > keys{{
+    1, 2, 3, 4, 5, 7, 9, 11, 22, 24, 44, 61, 66, 67 }};
+  return keys[ law.index() ];
```

Comment:
I sense some Austin influence here ... :-)

Reply:
This is one you can do using static constexpr functions on each LAW object as well but this works too.